packaging/install/020-ConfigMap.yaml

```diff
@@ -13,7 +13,7 @@ data:
     mqtt.host=0.0.0.0
     mqtt.port=1883
     #Apache Kafka common
-    kafka.bootstrap.servers=<kafka-bootstrap-servers>
+    kafka.bootstrap.servers=my-cluster-kafka-bootstrap:9092
```

Comment:
Was this intentional?

Reply:
Yes, it seems we have an "actual" hostname in other components like the canary for example.
We are using the kind of "dafault" from our examples having `my-cluster` as cluster name.

Reply:
Ok.